blobstore-s3/src/lib.rs

```diff
@@ -928,10 +928,8 @@ mod test {
     async fn aliases() {
         let mut map = HashMap::new();
         map.insert(format!("{}foo", ALIAS_PREFIX), "bar".to_string());
-        let ld = LinkDefinition {
-            values: map,
-            ..Default::default()
-        };
+        let mut ld = LinkDefinition::default();
```

Comment:
mutable data? outcast unclean!

Reply:
i know :( can't instantiate the struct without all fields